sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeries1Id.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is an array of 1 string value that uniquely identifies a Time Series instance.
+    /// </summary>
+    /// <remarks>
+    /// At the moment, Time Series Id service only supports string types to represent a TSI Id. In order to reduce the confusion on the user,
```

Comment:
**Time Series Insights or TSI** service

Reply:
class has been removed as we're trying a new design for TimeSeriesId :)